core_java/.gitignore

```diff
@@ -0,0 +1,9 @@
+target/
+*.class
+*.jar
+*.war
```

Comment:
you should have only one .gitignore which is stored in the root. please merge this file into the main .gitignore